MyPerfectOnboardingApplication/MyPerfectOnboarding.Api/Controllers/ListController.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Threading.Tasks;
+using System.Web.Http;
+using Microsoft.Web.Http;
+using MyPerfectOnboarding.Api.Models;
+
+namespace MyPerfectOnboarding.Api.Controllers
+{
+    [ApiVersion("1.0")]
+    [Route("api/v{version:apiVersion}/List")]
+    public class ListController : ApiController
+    {
+        public List<ListItem> items = new List<ListItem>
+        {
+            new ListItem(Guid.NewGuid(), "aaaa", false, DateTime.MinValue, DateTime.MinValue),
```

Comment:
After you remove the constructor from ListItem, you can easily change this to object initialization using { }.